<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>login_singup</class>
 <widget class="QMainWindow" name="login_singup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>505</width>
    <height>424</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(255, 40, 42, 240);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(41, 73, 185, 255), stop:0.865169 rgba(255, 39, 39, 255));
</string>
   </property>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>10</y>
      <width>391</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>311</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(123, 230, 50);
background-color: rgb(85, 0, 127);
font: 15pt &quot;MV Boli&quot;;
border-radius : 8px;
</string>
     </property>
     <property name="title">
      <string>sing in</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>291</width>
        <height>171</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Consol&quot;;
background-color: rgb(85, 0, 127);</string>
           </property>
           <property name="text">
            <string>username : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit">
           <property name="styleSheet">
            <string notr="true">font: 11pt &quot;Microsoft Sans Serif&quot;;
background-color: rgb(170, 0, 255);
border-radius: 8px;
</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font: 10pt &quot;Consol&quot;;
background-color: rgb(85, 0, 127);</string>
           </property>
           <property name="text">
            <string>password : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_2">
           <property name="styleSheet">
            <string notr="true">font: 11pt &quot;Microsoft Sans Serif&quot;;
background-color: rgb(170, 0, 255);
border-radius: 8px;</string>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>132</width>
        <height>22</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QRadioButton" name="radioButton">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;


</string>
         </property>
         <property name="text">
          <string>Customer</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioButton_2">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 8pt &quot;MS Shell Dlg 2&quot;;</string>
         </property>
         <property name="text">
          <string>Seller</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>135</x>
       <y>258</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="acceptDrops">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 12pt &quot;Segoe Print&quot;;
color: rgb(85, 255, 0);</string>
     </property>
     <property name="text">
      <string>log in</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>280</y>
       <width>231</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>9</number>
      </property>
      <property name="rightMargin">
       <number>19</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">
font: 57 10pt &quot;Mission Gothic Bold Italic&quot;;
color: rgb(255, 255, 127);</string>
        </property>
        <property name="text">
         <string>Don't have an account?</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">border-radius:8px;
color: rgb(255, 255, 0);
font: 75 12pt &quot;Segoe Print&quot;;
font: 57 11pt &quot;Mission Gothic Bold Italic&quot;;
background-color: transparent;
</string>
        </property>
        <property name="text">
         <string>sign up</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>505</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(41, 73, 185, 255), stop:0.865169 rgba(255, 39, 39, 255));</string>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
</string>
   </property>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(119, 54, 231);</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="exit"/>
  </widget>
  <action name="exit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resources/image/closewindow_hover.png</normaloff>:/resources/image/closewindow_hover.png</iconset>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
